portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -39,11 +44,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // Get response from the form
+    //Get response from the form
```

Comment:
there should be a space here between the // and the comment. revert this line.

Reply:
noted! updated!